src/github.com/getlantern/flashlight/server/server.go

```diff
@@ -295,6 +295,11 @@ func (server *Server) checkForBannedCountry(req *http.Request) error {
 }
 
 func (server *Server) lookupCountry(req *http.Request) (string, error) {
+	// Use the country CloudFlare gives us if it's available.
+	cf := req.Header.Get("Cf-Ipcountry")
```

Comment:
Just making sure -- this could be done independently of the country/ip lookup above, right?
Here we are just trying to get the information via Cloudflare's filter, as a shortcut.